### PR TITLE
DDF-04717 Remove mention of search templates from docs

### PR DESCRIPTION
**Pull request by rymach**
#### What does this PR do?
 Remove mention of search templates from docs

#### Who is reviewing it? 

@Schachte 
@Lambeaux 
#### Select relevant component teams: 
@codice/docs 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@AzGoalie

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4717 

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by rymach** [comment]
build now

File: distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc [hunk 1]
```diff
@@ -45,20 +45,6 @@ Before searching in ${branding}, at least one workspace must be created.
 .Start new workspace
 image::start-new-workspace.png[start new workspace icon]
```
Comment (mojogitoverhere) [inline distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc]: ❓ Should this line also be removed if we are removing `start-new-workspace.png`?

**Comment by mojogitoverhere** [comment]
@rymach Can you post a screenshot with your changes?

**Comment by rymach** [comment]
> @rymach Can you post a screenshot with your changes?

https://github.com/codice/ddf/blob/6f6cb583b576ed4a4ee988da24376289afc23f23/distribution/docs/src/main/resources/images/start-new-workspace.png

File: distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc (hunk 1 above)
Comment (rymach) [reply distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc]: I didn't remove - only edited https://github.com/codice/ddf/blob/6f6cb583b576ed4a4ee988da24376289afc23f23/distribution/docs/src/main/resources/images/start-new-workspace.png

**Comment by rymach** [comment]
@mojogitoverhere I was able to whip up a quick visual 

before
after

**Comment by mojogitoverhere** [comment]
Your changes look good.
Screen Shot 2019-04-18 at 12 12 36 PM

@ricklarsen While we are here, should we add a section about creating a new workspace with the `New Workspace` button?
Screen Shot 2019-04-18 at 12 14 12 PM

**Comment by rymach** [comment]
> @ricklarsen While we are here, should we add a section about creating a new workspace with the `New Workspace` button?

we can handle this new work in a new PR 👷

**Comment by ricklarsen** [comment]
> > @ricklarsen While we are here, should we add a section about creating a new workspace with the `New Workspace` button?
> 
> we can handle this new work in a new PR 👷 (it's not in the scope of the issue #4717)

i think that's a great idea

**Pull request merged**